build-tools/packages/build-cli/src/library/repoPolicyCheck/spacesOverTabsInYaml.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Handler, readFile } from "./common.js";
+
+/**
+ * Checks that *.yml/*.yaml files do not use tabs for indentation.
+ * Deliberately does not provide a resolver because automatic changes to yaml files, in particular those related to
+ * indentation, can be risky.
+ */
+export const handler: Handler = {
+	name: "indent-with-spaces-in-yaml",
+	match: /(^|\/)[^/]+\.ya?ml$/i,
```

Comment:
One instance of `/` is escaped, while the other isn't. IIRC `/`s don't need to be escaped, but we should probably be consistent.

Reply:
Good catch, I'll review it. Just copy pasted it from another handler 😄 .

Reply:
Running the regex in a tester, it seems like it needs to be escaped in the first capture scope, but not in the set. But escaping the second one for consistency seems like a good idea for legibility.

Reply:
Actually it seems to be correct. The first one needs escaping because otherwise JS thinks it's the delimiter that terminates the regex, but the one inside `[]` isn't interpreted as such even without escaping.

Reply:
The [better-regex linter rule](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/better-regex.md) actually wants to remove the escaping inside the character set. I won't fight it for now.